README.md

```diff
@@ -42,3 +56,11 @@ Also installed by Steam, so you may already have it configured. If not, create `
 
     # PS5 DualSense controller over bluetooth hidraw
     KERNEL=="hidraw*", KERNELS=="*054C:0CE6*", MODE="0660", TAG+="uaccess"
+
+### Credit
+
+nowrep for the original dualsensectl code
```

Comment:
Remove this please

Reply:
do you want me to remove the whole credit paragraph or only the line 62 ?

Reply:
Remove the line 62 (crediting me), you can keep the rest if you want.